examples/src/main/java/io/dapr/examples/OrderManager.java

```diff
@@ -0,0 +1,143 @@
+package io.dapr.examples;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpServer;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Redirect;
+import java.net.http.HttpClient.Version;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+import static java.lang.System.out;
+
+/**
+ * OrderManager web app.
+ * 
+ * Based on the helloworld Node.js example in https://github.com/dapr/samples/blob/master/1.hello-world/app.js
+ *
+ * To install jars into your local maven repo:
+ *   mvn clean install
+ *
+ * To run (after step above):
+ *   dapr run --app-id orderapp --app-port 3000 --port 3500 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.OrderManager
```

Comment:
If the app port here is 3000, the line:
```
 *   dapr run --grpc-port 3000 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.Example
```

Will cause a port collision

Reply:
Sorry, fixing it.

Reply:
Fixed the other one to be something else.